_posts/2019-11-27-gettingstarted-2020/getting-started-2020.Rmd

```diff
@@ -0,0 +1,144 @@
+
+The best-performing model is of little use if ephemeral. Straightforward ways of saving Keras models are explained in a dedicated [tutorial](https://tensorflow.rstudio.com/tutorials/beginners/basic-ml/tutorial_save_and_restore/). 
+
+<aside>
+Advanced users: Additional options exist; see the [tutorial on checkpoints](https://tensorflow.rstudio.com/guide/saving/checkpoints/).
+</aside>
+
+And unless one's just tinkering around, the question will often be: How can I deploy my model?
+There is a complete new section on [deployment](https://tensorflow.rstudio.com/deploy/), featuring options like `plumber`, Shiny, TensorFlow Serving and RStudio Connect.
+
+<aside>
+Check out the new section on [deployment options](https://tensorflow.rstudio.com/deploy/).
+</aside>
+
+After this workflow-oriented run-through, let's see about different types of data you might want to model.
+
+## Neural networks for different kinds of data
+
+No introduction to deep learning without image classification. The "Fashion MNIST" classification tutorial mentioned in the beginning is a good introduction, but it uses a fully connected neural network to make it easy to remain focused on the overall approach. Standard models for image recognition, however, are commonly based on a convolutional architecture. [Here](https://tensorflow.rstudio.com/tutorials/advanced/images/cnn/) is a nice introductory tutorial.
+
+For text data, the concept of _embeddings_ -- distributed representations endowed with a measure of similarity -- is central. As in the aforementioned text classification tutorial, embeddings can be learned using the respective Keras layer; in fact, the more idiosyncratic the dataset, the more recommendable this approach. Often though, it makes a lot of sense to use _pre-trained embeddings_, obtained from large language models trained on enormous amounts of data. With TensorFlow Hub, discussed in more detail in the last section, pre-trained embeddings can be made use of simply by integrating an adequate _hub layer_, as shown in [one of the Hub tutorials](https://tensorflow.rstudio.com/tutorials/beginners/basic-ml/tutorial_basic_text_classification_with_tfhub/).
+
+<aside>
+Models from TF Hub can now conveniently be integrated into a model as Keras layers.
+</aside>
+
+As opposed to images and text, "normal", a.k.a. _tabular_, a.k.a. _structured_ data often seems like less of a candidate for deep learning. Historically, the mix of data types -- numeric, binary, categorical --, together with different handling in the network ("leave alone" resp. embed) used to require a fair amount of manual fiddling. In contrast, the [Structured data tutorial](https://tensorflow.rstudio.com/tutorials/advanced/structured/classify/) shows the, quote-unquote, modern way, again using feature columns and feature specs. The consequence: If you're not sure that in the area of tabular data, deep learning will lead to improved performance -- if it's as easy as that, why not give it a try?
```

Comment:
There's another "resp." here where you'll want to flip it up e.g. ("leave alone" or embed, resp.)

Reply:
i think i don't like postfix resp... "or" should do as well ;-)